<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Draw the Circumcircle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Canvas" name="canvas" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_2"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionCircles"/>
    <addaction name="actionCenter"/>
    <addaction name="actionTriangles"/>
    <addaction name="actionServers"/>
    <addaction name="actionServer_routes"/>
    <addaction name="actionRoutes"/>
    <addaction name="actionShared_sides"/>
    <addaction name="actionNext_steps"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>Compile</string>
    </property>
    <addaction name="actionCheck_Delaunay"/>
    <addaction name="actionProcess_flip"/>
    <addaction name="actionVoronoi"/>
   </widget>
   <widget class="QMenu" name="menuEnable_Disable_drone_movement">
    <property name="title">
     <string>Drone movement</string>
    </property>
    <addaction name="actionEnable_Disable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDraw"/>
   <addaction name="menuAbout"/>
   <addaction name="menuCompile"/>
   <addaction name="menuEnable_Disable_drone_movement"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a file with triangles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCircles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circles</string>
   </property>
   <property name="toolTip">
    <string>Show Circles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Centers</string>
   </property>
   <property name="toolTip">
    <string>Show Centers</string>
   </property>
  </action>
  <action name="actionTriangles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Triangles</string>
   </property>
   <property name="toolTip">
    <string>Show triangles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCheck_Delaunay">
   <property name="text">
    <string>Check Delaunay</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionProcess_flip">
   <property name="text">
    <string>Process flip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionVoronoi">
   <property name="text">
    <string>Voronoi</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionLoad_Server_file">
   <property name="text">
    <string>Load Server file</string>
   </property>
  </action>
  <action name="actionLoad_Servers">
   <property name="text">
    <string>Load Servers</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionServers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Servers</string>
   </property>
   <property name="toolTip">
    <string>Show servers</string>
   </property>
  </action>
  <action name="actionRoutes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Routes</string>
   </property>
   <property name="toolTip">
    <string>Show routes</string>
   </property>
  </action>
  <action name="actionShared_sides">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shared sides</string>
   </property>
  </action>
  <action name="actionDrone_movements">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Drone movements</string>
   </property>
  </action>
  <action name="actionServer_routes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Server routes</string>
   </property>
  </action>
  <action name="actionNext_steps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Next steps</string>
   </property>
  </action>
  <action name="actionEnable_Disable">
   <property name="text">
    <string>Enable / Disable</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header location="global">canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
